File: 2547 - Roller Coaster.py
while True:
    count = 0
    try:
        temp_first_line = input()
        first_line = temp_first_line.split()
        n = int(first_line[0])
        a_min = int(first_line[1])
        a_max = int(first_line[2])

        for x in range(n):
            temp = int(input())
            if(temp >= a_min and temp <= a_max):
                count = count +1
        print(count)


    except EOFError:
        break

File: 2547 - Roller Coaster.c
#include <stdio.h>

int main()
{
    int i,n,temp,a_min,a_max, count;

    while(scanf("%d", &n) >= 0){
            count = 0;
        scanf("%d",&a_min);
        scanf("%d",&a_max);

        for(i=0;i<n;i++){
            scanf("%d",&temp);
            if(temp >= a_min && temp <= a_max){
                count++;
            }
        }
        printf("%d\n",count);
    }

    return 0;
}
